README.md

```diff
@@ -78,13 +78,13 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
 
-Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin).
 
-[![Run on Replit](https://replit.com/badge?caption=Import%20to%20Replit)](https://replit.com/github/upleveled/express-guest-list-api-memory-data-store)
+Change the `express-guest-list-api-memory-data-store` repo URL to include "box" (`githubbox.com` instead of `github.com`) and hit return. This will open the project on CodeSandbox.
 
-This will import the code from this repo to a new repl on Replit. Once it has been imported, run the application using the green **Run** button at the top center of the screen.
+Click the "Fork" button in the top right corner of the page to create your own personal copy of the project.
```

Comment:
we discussed some problems offline with accessing the devboxes created using these instructions:

1. `githubbox.com` does go to a URL with `/s/` in it (old sandbox URL)
   1. eg. https://githubbox.com/upleveled/express-guest-list-api-memory-data-store goes to https://codesandbox.io/s/github/upleveled/express-guest-list-api-memory-data-store
2. this `/s/` URL immediately redirects to a `/devbox/` URL (a "Synced Devbox")
   ![Screenshot 2024-05-13 at 12 08 27](https://github.com/upleveled/express-guest-list-api-memory-data-store/assets/1935696/5b352da8-d11f-446e-a5ca-9a5b0b48b973)
3. clicking on the Fork button at the top right then creates a "Draft Devbox"
   ![Screenshot 2024-05-13 at 11 58 07](https://github.com/upleveled/express-guest-list-api-memory-data-store/assets/1935696/e2e0e1fa-2465-4548-81bd-04d76349bafb)
4. attempting to visit the URL of the "Draft Devbox" API (eg. https://w3g6p5-4000.csb.app/) in an incognito browser leads to a "Log in" screen of CodeSandbox

a first idea would be to instruct users to then create a Shared Devbox from the Draft Devbox (add an extra instruction) - but this kind of defeats the purpose of using `githubbox` URLs - which is to keep the instructions simple and as close to 1-step instructions as possible

so let's try to find a simple way to create a Devbox with whatever sharing is needed

Reply:
I tried to find different solutions, to make it close to 1-step instructions. I didn't find a way without forking the Sandbox and setting the visibility to public. Added a step to the Readme
[bcc88b0](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/bcc88b0295837eed1a598c51af1cc09f5381227a)